src/components/ThemeToggle/ThemeToggle.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react'
+import styles from './ThemeToggle.module.scss'
+import { useAnalytics } from 'stores/analyticsContext'
+
+const ThemeToggle = () => {
+  // TODO: Replace this with the actual logic for changing the theme
+  const [theme, setTheme] = useState('light')
+  const { trackEvent } = useAnalytics()
+
+  const handleTheme = () => {
+    trackEvent(
+      'Dark mode',
+      'Click on light/dark mode toggle',
+      'Light/Dark mode toggle'
+    )
```

Comment:
Question: Can this call include what state was chosen? Like `Toggling to Dark mode` & `Toggling to Light mode`?

Reply:
Glad you brought this up. I had initially implemented it that way, but wasn't sure if we wanted that or not? If so, it's an easy change.

Reply:
Personally I would like to see that information too, but we can bring this up at sync and see what others would like.

Reply:
where did we land with this?

Reply:
Seems that we're holding off for now since we'll be introducing additional themes in the future.

Reply:
I think we talked about this at Sync when you were out, John was fine waiting for a more detailed event tracking at a later stage.